
----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------

----------------------header--------------------------
2018-04-14 17:05:46,385 [12] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 222
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)
2018-04-14 17:09:05,741 [17] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 222
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)
2018-04-14 17:11:10,225 [15] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 222
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)
2018-04-14 17:16:14,140 [30] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 222
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)
2018-04-14 17:21:04,844 [27] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 222
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)

----------------------footer--------------------------

----------------------header--------------------------
2018-04-14 17:25:12,689 [5] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 230
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)
2018-04-14 17:31:23,949 [10] ERROR log4netlogger - System.Data.Entity.Validation.DbEntityValidationException: 对一个或多个实体的验证失败。有关详细信息，请参见“EntityValidationErrors”属性。
   在 System.Data.Entity.Internal.InternalContext.SaveChanges()
   在 System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   在 System.Data.Entity.DbContext.SaveChanges()
   在 SS.Platform.OA.DalFactory.DbSession.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.DalFactory\DbSession.cs:行号 45
   在 SS.Platform.OA.BLL.BaseService`1.SaveChanges() 位置 f:\SS.Platform\SS.Platform.OA.BLL\BaseService.cs:行号 15
   在 SS.Platform.OA.UI.Portal.Controllers.EmployeeController.EditSave(Employee employee) 位置 j:\SS.Platform\SS.Platform.OA.UI.Portal\Controllers\EmployeeController.cs:行号 230
   在 lambda_method(Closure , ControllerBase , Object[] )
   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<InvokeActionMethodWithFilters>b__12()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodFilter(IActionFilter filter, ActionExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass15.<>c__DisplayClass17.<InvokeActionMethodWithFilters>b__14()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethodWithFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeAction(ControllerContext controllerContext, String actionName)

----------------------footer--------------------------

----------------------header--------------------------

----------------------footer--------------------------
